### ISSUE TITLE
Handbrake 1.5.1

### ISSUE THREAD

**Issue by mefbgil** [jlesage/docker-handbrake#204]
Hi @jlesage - First, I want to thank you for all the awesome work that you do, not only on this Docker image but the others you've build.  You've saved me tons of time and terabytes.  Really exceptional work.  QQ - do you have any plans to release an image with the new HB 1.5.1?  There are a lot of VAAPI optimizations promised and I'd curious about the speed and compression impacts that will come out of that.  Let us know and thanks again!

**Comment by jlesage** [comment]
Yes I started working on integrating the new version as soon as it came out.  Unfortunately, QSV no longer work for me.  It's not clear yet if it's because something is missing in the container or if because my CPU is no longer supported...

**Comment by mefbgil** [comment]
OK - is there anything I can do to help?  I'm running Docker 20.10.7 on Ubuntu 20.04.3 LTS on an Intel NUC Model NUC10i7FNB with an Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz.

**Comment by jlesage** [comment]
I made some progress, QSV is now properly detected, but an encoding fails for me.

Can you please try tag `v1.25.0-pre1` to see if it's working for you ?

**Comment by mefbgil** [comment]
Just tried it - the QSV options are not showing in the video encoder drop down.

**Comment by jlesage** [comment]
Can you try `v1.25.0-pre1` now?

**Comment by Irisheyes47** [comment]
Hi Jocelyn, I've just tried pre2, QSV options not showing in dropdown menu, as per @mefbgil 's comment re pre1.

Activity log states:
[12:00:53] CPU: 11th Gen Intel(R) Core(TM) i5-11500 @ 2.70GHz
[12:00:53]  - logical processor count: 12
[12:00:53] Intel Quick Sync Video support: no

Hope this helps

[Activity.log](https://github.com/jlesage/docker-handbrake/files/7954393/Activity.log)

**Comment by mefbgil** [comment]
Yep - same for me with the i7-10710U - no QSV options showing.  BTW I'm using the identical docker-compose.yml as I have been with the old HB image.

**Comment by Irisheyes47** [comment]
> Yep - same for me with the i7-10710U - no QSV options showing. BTW I'm using the identical docker-compose.yml as I have been with the old HB image.

Likewise, same docker-compose.yml...

**Comment by jlesage** [comment]
Ok, so I assume that there is nothing in the container's log indicating that QSV is not supported.

Can you try the following:

1. Re-create the container with privileged mode enabled.
2. Login to the container (`docker exec -ti <container name>`).
3. Install `strace`: `add-pkg strace`.
4. Share the output of the following command: `strace -f -e open,openat HandBrakeCLI`.

Thanks!

**Comment by Irisheyes47** [comment]
Hi Jocelyn

I'm using Portainer, how do I enable privileged mode?  Can't find an option!

**Comment by jlesage** [comment]
Under Capabilities, you can enable `SYS_PTRACE`.  This should be enough to run the strace tool.

**Comment by Irisheyes47** [comment]
Done, here's the output:
[strace.txt](https://github.com/jlesage/docker-handbrake/files/7954684/strace.txt)
 Also, the container log:
[_handbrake_latest_logs.txt](https://github.com/jlesage/docker-handbrake/files/7954695/_handbrake_latest_logs.txt)

**Comment by jlesage** [comment]
`v1.25.0-pre3` now available.
Please provide again the strike output (even if it's working).

**Comment by Irisheyes47** [comment]
Intel QSV shows up as an option now but none of the h265 options actually work, the same behaviour as version 1.24.2 unfortunately.

Strace output:
[Strace v1.25.0-pre3.txt](https://github.com/jlesage/docker-handbrake/files/7955677/Strace.v1.25.0-pre3.txt)

Encode logs:
[The Bourne Supremacy (2004) HEVC (1) 2022-01-28 17-22-37.log](https://github.com/jlesage/docker-handbrake/files/7955682/The.Bourne.Supremacy.2004.HEVC.1.2022-01-28.17-22-37.log)
[The Bourne Supremacy (2004) HEVC (1) 2022-01-28 17-22-54.log](https://github.com/jlesage/docker-handbrake/files/7955683/The.Bourne.Supremacy.2004.HEVC.1.2022-01-28.17-22-54.log)
[The Bourne Supremacy (2004) HEVC (1) 2022-01-28 17-23-09.log](https://github.com/jlesage/docker-handbrake/files/7955684/The.Bourne.Supremacy.2004.HEVC.1.2022-01-28.17-23-09.log)

Activity log:
[Activity.log](https://github.com/jlesage/docker-handbrake/files/7955692/Activity.log)

Hope this helps Jocelyn.

Pete

**Comment by wickedyoda** [comment]
Defently following this, I had started building an ubuntu GUI container and installing handbrake in it just for this, then found your project. The only issue I have with your setup over mine is performance. I can convert a 20gb file in about 2 hours vs yours of 13-15 hours on the same docker host.

**Comment by Irisheyes47** [comment]
From Handbrake (https://github.com/HandBrake/HandBrake/issues/4201), a fairly simple workaround from @heldiho53.

Adding lowpower=0 to VideoOptionExtra in the presets.json file makes the gui work. Just create a custom profile and add that option to the preset file. A good "so far" solution until something is implemented. 

I can confirm that it works, and I now have Intel QSV working perfectly.  I did have to restart the container.

@jlesage , I implemented this on your v1.25.0-pre3 testbuild.  I'll cross-post to the other issue.
